### ISSUE TITLE
Hanging on subscribeHtlcEvents()

### ISSUE THREAD

**Issue by MrHash** [RadarTech/lnrpc#76]
Not sure why but cannot await this endpoint on the router rpc, node just hangs. `subscribeInvoices()` on invoices rpc works fine. Please advise.

**Comment by cavanmflynn** [comment]
Hey @MrHash, what's your usage look like? Methods that return a stream shouldn't be awaited.

**Comment by MrHash** [comment]
Looks like this. The first stream works fine, the second hangs.

```
const invoicesSubscriber = await lnd.lnrpc.subscribeInvoices({});
  invoicesSubscriber
  .on('data', (invoice) => {
    publish(invoice);
    if (invoice.state === STATE_OPEN && !invoice.rPreimage.toString('hex')) {
      subscribe(invoice);
    }
  });

  const htlcSubscriber = await lnd.routerrpc.subscribeHtlcEvents({});
  htlcSubscriber
  .on('data', (event) => {
    console.log(event);
  });
```

if i don't use await on the second call i get the error `UnhandledPromiseRejectionWarning: TypeError: htlcSubscriber.on is not a function`

**Comment by cavanmflynn** [comment]
Thanks, looking into this.

**Comment by cavanmflynn** [comment]
@MrHash A fix has been published in [v0.10.0-beta.0](https://www.npmjs.com/package/@radar/lnrpc/v/0.10.0-beta.0)

**Comment by MrHash** [comment]
Excellent, working fine now.
